packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/dedicatedCloud-user.html

```diff
@@ -165,6 +165,14 @@
             <span data-translate="common_no" data-ng-if="!$row.nsxRight">
             </span>
         </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'dedicatedCloud_USER_encryption' | translate"
+        >
+            <span data-translate="common_yes" data-ng-if="$row.encryptionRight">
```

Comment:
Would it be possible to add this translation here https://github.com/ovh/manager/blob/ec7b96a67adc8a61d1158f8b7f22cb9baef2ad65/packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/translations/Messages_fr_FR.json instead  of using commons ?

Reply:
Sure, will do.
I am not really aware of your coding rules so I just checked how you did for other fields nearby and did the same.

Reply:
Done and enerything is still working (but will require more translations)
![Capture d’écran 2022-07-19 à 14 54 53](https://user-images.githubusercontent.com/666182/179756375-c3e6c170-f710-4f29-bed5-0ef39be2575a.png)